sky/dist/BUILD.gn

```diff
@@ -7,6 +7,7 @@ root_dist_dir = "$root_build_dir/dist"
 copy("sky_viewer") {
   sources = [
     "$root_build_dir/sky_viewer.mojo",
+    "$root_build_dir/symbols/libsky_viewer_library.so",
```

Comment:
This breaks 'gn gen --check' (which is what sky/tools/gn does) on mac.  On mac, dynamic libraries have the suffix 'dylib' not 'so'.